impeller/geometry/path.cc

```diff
@@ -245,7 +245,7 @@ Path::Polyline Path::CreatePolyline(Scalar scale) const {
 
   auto get_path_component =
       [this](size_t component_i) -> std::optional<const PathComponent*> {
-    if (component_i >= components_.size()) {
+    if (component_i < 0 || component_i >= components_.size()) {
```

Comment:
component_i is unsigned. Is this necessary?

Reply:
My bad.  Should all be `size_t` instead of `int`. Done ^_^